### PR TITLE
Merging rafaelvfalc/asperathos-visualizer multi-cluster branch in ufcg/asperathos-visualizer master branch

### PR DESCRIPTION
**Pull request by rafaelvfalc**
* Starting unit tests implementations for the component.
* Adding the functionality of add/delete/active a specific cluster in a Asperathos Visualizer instance execution time.
* Updating documentation with new endpoints.

### CODE REVIEW

**Review by gabrielsvinha — CHANGES_REQUESTED** [review]
Fix conflicts with master branch.

**Comment by rafaelvfalc** [comment]
The PR is able to 'Squash and merge' with the master branch. It is not the best approach? In this way all five commits will be combined in one commit in the base branch.

File: visualizer/service/api/v10.py [hunk 1]
```diff
@@ -13,6 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+import filecmp
```
Comment (gabrielsvinha) [inline visualizer/service/api/v10.py]: Are these imports used somewhere?

Comment (rafaelvfalc) [reply visualizer/service/api/v10.py]: Not, both imports should be removed

File: visualizer/service/api/__init__.py [hunk 2]
```diff
@@ -36,9 +37,18 @@
 
     """ Grafana parameters """
     if 'k8s-grafana' in plugins:
-        k8s_conf_path = config.get('k8s-grafana', 'k8s_conf_path')
+
+        # Setting default value
+        k8s_conf_path = "./data/conf"
```
Comment (clenimar) [inline visualizer/service/api/__init__.py]: can you please define it in a constant?

File: visualizer/service/api/__init__.py [hunk 3]
```diff
@@ -60,9 +70,34 @@
...
+""" Gets the IP address of one a the node contained
+    in a Kubernetes cluster
+
+    Raises:
+        Exception -- It was not possible to connect with the
+        Kubernetes cluster.
+
+    Returns:
+        string -- The node IP
+    """
```
Comment (clenimar) [inline visualizer/service/api/__init__.py]: nit: indentation issues and an unnecessary blank line between docstring and method it describes.

File: visualizer/service/api/v10.py [hunk 4]
```diff
@@ -13,6 +13,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
```
Comment (clenimar) [inline visualizer/service/api/v10.py]: you don't seem to be using `os` anywhere here.

**Review by clenimar — APPROVED** [review]
thanks for the PR, this is a great addition to the codebase. just pointed a few small issues inline, but they're not really blockers and can be addressed in follow-up commits.

File: visualizer/service/api/v10.py (hunk 4 above)
Comment (rafaelvfalc) [reply visualizer/service/api/v10.py]: Modification made in commit 5cddb79d05f7fb24c82ab6018736cd0ab7adfd40

File: visualizer/service/api/__init__.py (hunk 3 above)
Comment (rafaelvfalc) [reply visualizer/service/api/__init__.py]: Modification made in commit 5cddb79d05f7fb24c82ab6018736cd0ab7adfd40

File: visualizer/service/api/__init__.py (hunk 2 above)
Comment (rafaelvfalc) [reply visualizer/service/api/__init__.py]: Modification made in commit 5cddb79d05f7fb24c82ab6018736cd0ab7adfd40

**Review by gabrielsvinha — APPROVED** [review]
topzera

**Pull request merged**